PermanentOrg/web-app
# Remove TSlint

TSLint is deprecated, yet still was a dependency in our project with separate configuration files. We previously installed eslint, and as a result TSLint is no longer even used. Get rid of it.
    
TSLint also used codelyzer as a dependency which is also deprecated. Codelyzer includes some vulnerable dependencies, such as importing very old versions of Angular. Remove codelyzer as well to fix some security alerts.

Resolves PER-8557.
Also resolves [this Dependabot alert](https://github.com/PermanentOrg/web-app/security/dependabot/63)

**Steps To Test:**
1. Run `npm install`
2. Run `npm run lint`
3. You should see a lot of linting errors since we haven't disabled eslint's rules, but you *shouldn't* see any actual failures related to the removal of TSLint or Codelyzer.
